File: CS/DesignerSample/Form1.cs
using DevExpress.XtraCharts;
using MultiPaneExtension;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DesignerSample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            MultiPaneModule module = new MultiPaneModule();
            module.Attach(dashboardDesigner1);

            dashboardDesigner1.LoadDashboard(@"Data/MultiPaneCharts.xml");
        }
    }
}

File: VB/DesignerSample/Form1.vb
Imports DevExpress.XtraCharts
Imports MultiPaneExtension
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace DesignerSample
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			dashboardDesigner1.CreateRibbon()
			Dim [module] As New MultiPaneModule()
			[module].Attach(dashboardDesigner1)

			dashboardDesigner1.LoadDashboard("Data/MultiPaneCharts.xml")
		End Sub
	End Class
End Namespace
